resources/lib/Plugin.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+class Plugin
+{
+    const PLUGIN_NAMESPACE = 'Findologic';
+    const PLUGIN_IDENTIFIER = 'findologic-plugin-api';
+    const PLUGIN_VERSION = '5.1.1';
```

Comment:
Why does this class exist, when this was is already used:
https://github.com/findologic/plugin-plentymarkets-ceres-api/blob/develop/src/Constants/Plugin.php